src/main/scala/viper/gobra/frontend/Desugar.scala

```diff
@@ -3329,10 +3497,10 @@ object Desugar {
     }
 
     /** returns the relativeId with the CONTINUE_LABEL_SUFFIX appended */
-    def continueLabel(loop: PForStmt, info: TypeInfo) : String = relativeId(loop, info) + CONTINUE_LABEL_SUFFIX
+    def continueLabel(loop: PNode, info: TypeInfo) : String = relativeId(loop, info) + CONTINUE_LABEL_SUFFIX
```

Comment:
why did you make the type of loop so much weaker? is it to be able to deal with range and normal loops? If so, wouldn't it make more sense to have a trait that both types of loops implement, and require this param to be of that trait type?

Reply:
Fixed for the functions continueLabel and breakLabel, left it as is for relativeId since it is used by outline too and it seems like something generally useful